src/common/webrequest.cpp

```diff
@@ -636,6 +636,14 @@ void wxWebResponseImpl::Init()
 }
 
 wxString wxWebResponseImpl::GetMimeType() const
+{
+    wxString contentType( GetContentType() );
+    const auto separatorPosition = contentType.find(';');
+    return (separatorPosition != wxString::npos) ?
+        contentType.substr(0, separatorPosition) : contentType;
```

Comment:
In general I'm for using the standard string functions, but in this particular case all this could be replaced with just
```suggestion
    return GetContentType().BeforeFirst(';');
```
which seems worth it as it's shorter and much more readable.

Reply:
Totally fine with me.  I was just trying to use the more "std::string-like" API, a habit I have gotten into.  But yes, this is much easier to read.